src/plugins/customBorders/customBorders.js

```diff
@@ -133,68 +141,147 @@ class CustomBorders extends BasePlugin {
   }
 
   /**
-   * Get index of border from the settings.
-   *
-   * @param {String} className Class name as string.
-   * @returns {Number}
-   */
-  getSettingIndex(className) {
-    let index = -1;
+    * Set custom borders.
```

Comment:
As this method is public and will be visible in the docs could you add some more info about using this method with some examples?